cmd/service/utils.go

```diff
@@ -77,40 +75,17 @@ func downloadServiceIfNeeded(path string) (newPath string, didDownload bool, err
 	if !govalidator.IsURL(path) {
 		return path, false, nil
 	}
-	newPath, err = createTempFolder()
+	newPath, err = ioutil.TempDir("", "mesg-")
 	if err != nil {
 		return "", false, err
 	}
-	if err := gitClone(path, newPath, "Downloading service..."); err != nil {
+	fmt.Println("Downloading service...")
```

Comment:
forgot to use _utils.ShowSpinnerForFunc()_? it was defined in previous version of _gitClone()_